lib/common/grpc-service.js

```diff
@@ -332,24 +364,20 @@ GrpcService.convertValue_ = function(value) {
     convertedValue = {
       structValue: GrpcService.objToStruct_(value)
     };
-  } else if (is.date(value)) {
```

Comment:
According to this comment (https://github.com/GoogleCloudPlatform/gcloud-node/pull/1374#issuecomment-227599140), it doesn't sound like it.